web/app/scripts/saved-filters/saved-filters-modal-controller.js

```diff
@@ -0,0 +1,63 @@
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function SavedFiltersModalController($modalInstance, $rootScope,
+                                         FilterState, Notifications, SavedFilters) {
+        var ctl = this;
+        ctl.label = '';
+        ctl.save = save;
+
+        // linter complains if this function is defined below for some reason
+        ctl.close = function() { $modalInstance.close(); };
```

Comment:
That's weird; if it was defined the same ways as `save` is I don't see why there would be a problem. Does changing the name of the function (maybe `closeModal`) help?